src/flows/pipes/RawFluxEditor/readOnly.tsx

```diff
@@ -47,4 +50,8 @@ const Query: FC<PipeProp> = ({Context}) => {
   )
 }
 
-export default Query
+export default Props => (
+  <VariablesProvider>
```

Comment:
move this to the pipe component instead of the individual pipe expression so that we can give a centralized location for future expansion of the variable concept